src/AttributeTranslator/ArrayAttributeTranslator.php

```diff
@@ -11,6 +11,10 @@
  */
 final class ArrayAttributeTranslator implements AttributeTranslatorInterface
 {
+    /**
+     * @param array $translations Translations array where each key is an attribute name and the corresponding name is
```

Comment:
Type `array` is less specific + it's an extra copy. Suggest to move it to "var".